tests/acceptance/tour-test.js

```diff
@@ -24,8 +23,6 @@ module("Acceptance | tour", function (hooks) {
   test("shows a welcome dialog", async function (assert) {
     await visit("/");
 
-    await waitForStep();
-
```

Comment:
Why deleting this?

Reply:
because it's a part of `ember-site-tour/test-support/helpers`, and ember site tour is no more exists

Reply:
Ah sorry, did confuse it with the *standard* ember test helpers.